src/explore-education-statistics-common/src/modules/charts/util/getDataSetCategoryConfigs.ts

```diff
@@ -16,11 +17,19 @@ import { colours } from '@common/modules/charts/util/chartUtils';
 import expandDataSet from '@common/modules/charts/util/expandDataSet';
 import generateDataSetKey from '@common/modules/charts/util/generateDataSetKey';
 import generateDefaultDataSetLabel from '@common/modules/charts/util/generateDefaultDataSetLabel';
+import { Filter } from '@common/modules/table-tool/types/filters';
 import { FullTableMeta } from '@common/modules/table-tool/types/fullTable';
 import { Dictionary } from '@common/types';
 import keyBy from 'lodash/keyBy';
 import omit from 'lodash/omit';
 import uniqBy from 'lodash/uniqBy';
+import groupBy from 'lodash/groupBy';
+
+interface DeprecatedGrouping {
+  customGroups: [];
```

Comment:
Just wondering if this could be typed a bit more strictly? Assuming we don't know the exact shape of customGroups so don't think it's a bit deal if that's the case

Reply:
yep, it can be, though I also realised I was duplicating an existing type so have removed this.